src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="HOME">
+        <img
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1715558400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=IeI5VLwaCRBGI9mZTqEVPjJBQg5JW5NArUpF5mv7jTetxc5xh0hzQ2ZQ8PqPAb3dlZxW5Jjm5GvIK~331JImiSkB81jJ8wLgrDCksYwCh3ZDtYGEDKYaSP-HAd6mbMJu-GqUFAcKU3RyAhlbn4aazhEEOR9Qk5tEExvh69eunqSQdFPrNLkefx0Qu-SWEgYx~KnNMWBZuAVD4y0diX008eVKHOlYipMlGBAAnddSguaRxx4p~RHXwIuBC4fh50Ypq0qezyhXGQRPgHqnS5D85797Xce6d5oDcH2xlQhOrh3f-h5c0OnEY94DCCLjHkcyEfkpBZIGH6vI5ZNqYb-Nxg__"
+          alt="logo.img"
+        />
+      </a>
+
+      <nav>
+        <ul class="noPadding">
+          <a
+            class="link is-active"
+            href="APPLE"
+          >
+            APPLE
+          </a>
+        </ul>
+
+        <ul>
+          <a
+            class="link"
+            href="SAMSUNG"
+          >
+            SAMSUNG
+          </a>
+        </ul>
```

Comment:
Let's create one list(ul) and use `li` tags for it`s children, that is how lists work
```suggestion
        <ul class="noPadding">
         <li class="list__item"> 
           <a class="link is-active" href="APPLE">
              APPLE
            </a>   
          </li> 
          <li class="list__item">
            <a class="link" href="SAMSUNG">
              SAMSUNG
           </a>
          </li>
        </ul>

```